subworkflows/local/vcf_concordance_glimpse2/main.nf

```diff
@@ -6,8 +6,8 @@ include { GUNZIP                      } from '../../../modules/nf-core/gunzip'
 workflow VCF_CONCORDANCE_GLIMPSE2 {
 
     take:
-        ch_vcf_emul   // VCF file with imputed genotypes [ [id], vcf, csi]
-        ch_vcf_truth  // VCF file with truth genotypes   [ [id], vcf, csi]
+        ch_vcf_emul   // VCF file with imputed genotypes [ [id, panel, tool, chr], vcf, csi]
```

Comment:
We have to take into account that it won't have a tool (unless we require them to do so) when users start from `--step validate`.

Reply:
That's true. Normally the subworkflow does not depend of its presence in the meta map.
We should find a way to specify if it's optional or not in the comment.